components/Hero.tsx

```diff
@@ -34,7 +35,17 @@ export const Hero: React.FC<HeroProps> = ({ children, topAction = null }) => {
     if (!selectedSynth) {
       return;
     }
-    router.push(`/${selectedSynth.address}`);
+    if (selectedSynth.externalUrl) {
+      const newWindow = window.open(
+        selectedSynth.externalUrl,
+        "_blank",
+        "noopener,noreferrer"
+      );
+      if (newWindow) newWindow.opener = null;
+    } else {
+      const chainName = chainIdToNameLookup[selectedSynth.chainId];
```

Comment:
id suggest falling back to chainId in the case the lookup fails, just so we have something than undefined and could debug a bit easier